compress error msg for run_all_miracl

crystina-z opened the pull request:
For issue #1325

lintool reviewed (changes requested):
Hi @crystina-z I just realized - this won't work for the dev bindings that we have.

Right now, we can do `--topics miracl-v1.0-ar-dev` and it'll work.

With your path, I think the code will throw an error?

crystina-z commented:
right but the `run_all_miracl.py` would only read in `miracl.yaml`, which writes the topic path

crystina-z commented:
```
condition bm25.ar:                                                              
  - split: train          
    Running: python -m pyserini.search.lucene --language ar --topics miracl-v1.0-ar-train --index miracl-v1.0-ar --output runs/run.miracl.bm25.ar.train.txt --batch 128 --threads 16 --bm25 --hits 100
Skip miracl-v1.0-ar-train.                                                      
  - split: dev            
      nDCG@10: 0.4809 [OK]
      R@100  : 0.8885 [OK]   
```

^^^ current output

lintool commented on scripts/repro_matrix/run_all_miracl.py:
```diff
@@ -70,12 +77,17 @@ def print_results(metric, split):
+                    stderr = rtn.stderr.decode()
+                    topic_fn = extract_topic_fn_from_cmd(cmd)
+                    if f"ValueError: Topic {topic_fn} Not Found" in stderr:
+                        print(f"Skip {topic_fn}.")
```
```suggestion
                        print(f"Skipping {topic_fn}: file not found.")
```

lintool replied:
Also, change to single quotes?

lintool reviewed (approved):
Address minor nits and go ahead and merge please.
